code/game/area/Space Station 13 areas_vr.dm

```diff
@@ -1,5 +1,14 @@
 //TFF 28/8/19 - cleanup of areas placement - removes all but rogueminer_vr stuff.
-
+/area
+	var/limit_mob_size = TRUE //If mob size is limited in the area.
+
+/area/Entered(mob/living/H, oldLoc)
+	//Clamps mob size when entering a new area that has size limit.
+	//Mobs with size_uncapped set to TRUE or 1 will be ignored.
+	. = ..()
+	if (src.limit_mob_size && !H.size_uncapped)
+		H.resize(H.size_multiplier)
```

Comment:
Alright so to answer this one! First, been a bit since I worked with DM code again, so did that casting out of habit.
Now, in regards to overhead, it may have some overhead, but it's argueably better than having it in life() proc, where it gets called far more often, wasn't it specifically once per tick?